x-pack/plugins/enterprise_search/common/types/indices.ts

```diff
@@ -18,6 +18,7 @@ import { Crawler } from './crawler';
 export interface ElasticsearchIndex {
   count: number; // Elasticsearch _count
   health?: HealthStatus;
+  hidden?: boolean;
```

Comment:
suggestion(non-blocking): This should probably just be a mandatory property (see below).